sagemaker-debugger/build_your_own_container_with_debugger/docker/tf_keras_resnet_byoc.py

```diff
@@ -0,0 +1,110 @@
+"""
+This script is a ResNet training script which uses Tensorflow's Keras interface, and acts as an
+example of how to use SageMaker Debugger if you use either your own custom container to run in SageMaker, or your own script outside SageMaker.
```

Comment:
Change "if you use either your..." to "when you use your own custom container in SageMaker or your own script outside SageMaker."